### DISCUSSION TITLE
I can't play witm this mod

### DISCUSSION THREAD

**Discussion by BlueCapybara** [FabricMC/fabric-api#2883] [Player support]
i have a fabric api on 1.18.2, i have fabric on 1.18.2 i instal a mod on 1.18.2 and i can't play this mod:

Incompatible mod set!
net.fabricmc.loader.impl.FormattedException: Mod resolution encountered an incompatible mod set!
A potential solution has been determined:
	 - Install fabric-language-scala, any version.
Unmet dependency listing:
	 - Mod 'Fluid Physics' (fluidphysics) 1.7.0+fabric-1.18.2 requires any version of fabric-language-scala, which is missing!
	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:190)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:148)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

pls help, if this is hard, please write me on my discord: Bluecapybara#2409

**Comment by haykam821** [comment]
This error indicates that the Fluid Physics mod has an unmet dependency on [Fabric Language Scala](https://curseforge.com/minecraft/mc-mods/fabric-language-scala). You must download and install Fabric Language Scala for this mod to work.
